term/src/test/mod.rs

```diff
@@ -141,10 +141,19 @@ impl TestTerm {
         self.print(format!("{}", osc));
     }
 
+    fn hyperfile(&mut self, file: &Arc<Hyperfile>) {
+        let osc = OperatingSystemCommand::SetHyperfile(Some(file.as_ref().clone()));
+        self.print(format!("{}", osc));
+    }
+
     fn hyperlink_off(&mut self) {
         self.print("\x1b]8;;\x1b\\");
     }
 
+    fn hyperfile_off(&mut self) {
+        self.print("\x1b]8;;\x1b\\");
```

Comment:
8 vs 88

Reply:
Good catch if I am using 88 then it should be 88, right. Unless you know of a better number?

Reply:
done